pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -212,8 +212,8 @@ func (l *LibvirtConnection) ListAllDomains(flags libvirt.ConnectListAllDomainsFl
 		return nil, err
 	}
 	doms := make([]VirDomain, len(virDoms))
-	for i, d := range virDoms {
-		doms[i] = &d
+	for i := range virDoms {
+		doms[i] = &virDoms[i]
```

Comment:
I find this type of fix far more readable than introducing a new variable with an underscored_ name.

Reply:
Not sure why. This is the common expression of taking the address of an array  component.

Reply:
I was probably unclear. I find

```
for i := range virDoms {
		doms[i] = &virDoms[i]
```
far clearer than the
```
for i, d_ := range virDoms {
                d := d_
		doms[i] = &d
```
you have elsewhere.

Reply:
I understood and answered accordingly ...

Reply:
changed to use the array in most cases